ui/js/filters.js

```diff
@@ -189,3 +190,11 @@ treeherder.filter('absoluteValue', function () {
         return Math.abs(input);
     };
 });
+
+treeherder.filter('reduceNumberLength', function () {
```

Comment:
let's call this filter `abbreviatedNumber` (the convention for filters seems to be naming them after the thing the input is being transformed to, rather than the process)